apps/web/app/(app)/[emailAccountId]/assistant/RuleForm.tsx

```diff
@@ -1226,6 +1240,68 @@ function ActionCard({
           {/* Show the variable pro tip when action has visible fields that can use variables */}
           {shouldShowProTip && <VariableProTip />}
 
+          {/* Show delay toggle and input for all actions that support delays */}
+          {canActionBeDelayed(action.type) && (
+            <div className="mt-4">
+              <div className="flex items-start justify-between">
+                <div className="flex-1">
+                  {/* Show delay input controls when delay has been enabled */}
+                  {watch(`actions.${index}.delayInMinutes`) !== null && (
+                    <DelayInputControls
+                      index={index}
+                      delayInMinutes={watch(`actions.${index}.delayInMinutes`)}
+                      setValue={setValue}
+                    />
+                  )}
+                </div>
+
+                <div className="flex items-center space-x-2">
+                  <TooltipExplanation
+                    text="Schedule this action to execute after a delay from when the email was received. Useful for follow-ups, reminders, or giving senders time to respond."
```

Comment:
Core use case would delayed archive. so think we should adjust this copy a bit.

Reply:
"For example, archive an email a week after receiving it" instead of `Useful for...`